### DISCUSSION TITLE
What is the use-case for the <c-component> with the "is" attribute?

### DISCUSSION THREAD

**Discussion by ra-dave** [wrabit/django-cotton#108] [Q&A] [answered]
I've been creating dynamic components in a {% for.... %) loop just fine by using the component name directly in the for loop. 

When would I want to use `<c-component>` with "is" attribute instead?

**Comment by wrabit** [comment]
Show me the code you are using to do as you say.

**Comment by wrabit** [comment, marked as answer]
@ra-dave I don't have to reach for it very often, but there are times. An example:

```python
# context
{'icons': [ 'pencil', 'pencil-square', 'pencil-circle', ... ] }
```

```html
<!-- icons are saved in: 
  cotton/pencil.html
  cotton/pencil_square.html
  ... 
-->
{% for icon in icons %}
    <c-component :is="icon" />
{% endfor %}
```

Or they might be in subfolders:

```html
{% for icon in icons %}
    <c-component is="subfolder.{{ icon }}" />
{% endfor %}
```
